package.json

```diff
@@ -72,7 +72,7 @@
     "@babel/plugin-transform-flow-strip-types": "^7.8.3",
     "@babel/plugin-transform-react-jsx": "^7.8.3",
     "@babel/plugin-transform-runtime": "^7.8.3",
-    "@babel/preset-env": "^7.8.7",
+    "@babel/preset-env": "^7.10.4",
```

Comment:
This should have also updated `package-lock.json` but I don't see it here? 🤔

Reply:
Yep, forgot to push it